dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:latest
+COPY ./ /feathr
+
+RUN apt-get update && \
+    apt-get install -y \
+    build-essential python3.8 python3.8-dev python3-pip git 
+
+RUN python3.8 -m pip install pip --upgrade
+RUN python3.8 -m pip install wheel
+EXPOSE 80
+
+WORKDIR /feathr/feathr_project/feathr/api
```

Comment:
change the folder name to feature_registry_api

Reply:
@blrchen will fix these docker-related modifications later